x-pack/plugins/canvas/public/lib/legend_options.js

```diff
@@ -4,25 +4,37 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 export const legendOptions = [
   {
```

Comment:
Please check that when we use for example German locale it works. We have issues when translate constants. If it does not work for German locale then we need to use functions instead of variables